docs/user/kafka/confluent.rst

```diff
@@ -7,11 +7,189 @@ Confluent Integration
+Value       Description
+=========== ==============================================================
+``4326``    Longitude/latitude coordinate system on the WGS84 ellipsoid
+=========== ==============================================================
+
+``geomesa.date.format``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Indicates that the field should be interpreted as a ``Date`` in the given format.
+
+=========================== ===================== ====================================================
+Value                       Schema Field Type     Description
+=========================== ===================== ====================================================
+``epoch-millis``            ``LONG``              Milliseconds since the Unix epoch
+``iso-date``                ``STRING``            Date format ``yyyy-MM-dd``
+``iso-instant``             ``STRING``            Date format ``yyyy-MM-dd'T'HH:mm:ss.SSSZZ``
+``iso-instant-no-millis``   ``STRING``            Date format ``yyyy-MM-dd'T'HH:mm:ssZZ``
+=========================== ===================== ====================================================
+
+``geomesa.date.default``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Indicates that the field represents the default ``Date`` for this ``SimpleFeatureType``. If the key
+``geomesa.date.format`` is not present on the same schema field, this attribute will be ignored.
+
+=========== =============================
+Value       Description
+=========== =============================
+``true``    The default ``Date``
+``false``   Not the default ``Date``
+=========== =============================
+
+``geomesa.index``
+^^^^^^^^^^^^^^^^^
+
+Indicates that the field should be indexed by GeoMesa.
+
+=========== ==================================
+Value       Description
+=========== ==================================
+``true``    Index this attribute
+``false``   Do not index this attribute
+``full``    Index the full ``SimpleFeature``
+=========== ==================================
+
+``geomesa.cardinality``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Specifies a cardinality hint for the attribute, for use by GeoMesa in query planning. This property should be used
+in conjunction with ``geomesa.index``.
+
+=========== ====================================
+Value       Description
+=========== ====================================
+``high``    Prioritize this attribute index
+``low``     De-prioritize this attribute index
+=========== ====================================
+
+``geomesa.avro.visibility.field``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specifies a field in the Avro schema which should be used as the visibility for features of the ``SimpleFeatureType``.
+
+============= ===================== ========================================================
+Value         Schema Field Type     Description
+============= ===================== ========================================================
+``true``      ``STRING``            Use the value of this field as the feature visibility
```

Comment:
Ok, so first read through....  is it an error to have multiple attributes with this set?

Reply:
Not necessarily an error but it might not do what the user expects. I just have it as the last one in the schema will be used right now

Reply:
There's now an error if you have multiple.